docs/en/usage.rst

```diff
@@ -0,0 +1,74 @@
+Usage
+=====
+
+Command Line
+------------
+
+Access the Doctrine command line as following
+
+.. code:: sh
+
+   php public/index.php
+
+or
+
+.. code:: sh
+
+   ./vendor/bin/doctrine-module
+
+Service Locator
```

Comment:
`Service Manager`

Service locator is no longer in the vernacular for this context.